### ISSUE TITLE
AttributeError: Parameter not found: sewage_water_district when running sewage-potential tool

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CityEnergyAnalyst#2213]
When running `cea sewage-potential`, the following error occurs:

```
Workflow step: script=sewage-potential
Running sewage_potential with args {'sewage_water_ratio': 0.95, 'heat_exchanger_length': 120, 'scenario': 'C:\\reference-case-open\\baseline'}
City Energy Analyst version 2.18.1
Running `cea sewage-potential` with the following parameters:
- general:scenario = C:\reference-case-open\baseline
  (default: C:\reference-case-open\baseline)
- sewage:heat-exchanger-length = 120
  (default: 120)
- sewage:sewage-water-ratio = 0.95
  (default: 0.95)
DEBUG:fiona:Creating a not-responsible GDALEnv in drivers()
DEBUG:Fiona:GDAL_DATA: C:\ProgramData\CityEnergyAnalyst\Dependencies\Python\Library\share\gdal
DEBUG:Fiona:PROJ_LIB: C:\ProgramData\CityEnergyAnalyst\Dependencies\Python\Library\share
DEBUG:Fiona:GDAL_DATA: C:\ProgramData\CityEnergyAnalyst\Dependencies\Python\Library\share\gdal
DEBUG:Fiona:PROJ_LIB: C:\ProgramData\CityEnergyAnalyst\Dependencies\Python\Library\share
DEBUG:Fiona:Got coordinate system
DEBUG:Fiona:Got coordinate system
DEBUG:Fiona:Index: 0
DEBUG:fiona.collection:Flushed buffer
DEBUG:fiona.collection:Stopped session
Traceback (most recent call last):
  File "C:\ProgramData\CityEnergyAnalyst\Dependencies\Python\Scripts\cea-script.py", line 11, in <module>
    load_entry_point('cityenergyanalyst', 'console_scripts', 'cea')()
  File "c:\programdata\cityenergyanalyst\cityenergyanalyst\cea\interfaces\cli\cli.py", line 64, in main
    script_module.main(config)
  File "c:\programdata\cityenergyanalyst\cityenergyanalyst\cea\tests\workflow.py", line 45, in main
    do_script_step(config, step)
  File "c:\programdata\cityenergyanalyst\cityenergyanalyst\cea\tests\workflow.py", line 112, in do_script_step
    run(config, py_script, **py_parameters)
  File "c:\programdata\cityenergyanalyst\cityenergyanalyst\cea\tests\workflow.py", line 29, in run
    f(config=config, **kwargs)
  File "c:\programdata\cityenergyanalyst\cityenergyanalyst\cea\api.py", line 32, in script_runner
    script_module.main(config)
  File "c:\programdata\cityenergyanalyst\cityenergyanalyst\cea\resources\sewage_heat_exchanger.py", line 185, in main
    calc_sewage_heat_exchanger(locator=locator, config=config)
  File "c:\programdata\cityenergyanalyst\cityenergyanalyst\cea\resources\sewage_heat_exchanger.py", line 43, in calc_sewage_heat_exchanger
    V_lps_external = config.sewage.sewage_water_district
  File "c:\programdata\cityenergyanalyst\cityenergyanalyst\cea\config.py", line 235, in __getattr__
    raise AttributeError("Parameter not found: %s" % item)
AttributeError: Parameter not found: sewage_water_district
```

This is probably due to a bad configuration of `scripts.yml`

**Comment by JIMENOFONSECA** [comment]
@daren-thomas this is being solved in my work on the optimization script

**Comment by daren-thomas** [comment]
@JIMENOFONSECA how tightly is it coupled with your optimization script? would it be possible to break it out into a separate branch so we can merge it asap?

**Comment by JIMENOFONSECA** [comment]
it is!, I am working in the details, a PR should be ready by friday…

_______________________

Dr. Jimeno A. Fonseca
Senior researcher and project coordinator
Future cities Laboratory
Singapore ETH-Centre

Chair of Architecure and Building Systems
ETH Zurich

On 29 Jul 2019, 5:55 PM +0800, Daren Thomas <notifications@github.com>, wrote:

@JIMENOFONSECA<https://github.com/JIMENOFONSECA> how tightly is it coupled with your optimization script? would it be possible to break it out into a separate branch so we can merge it asap?

**Comment by JIMENOFONSECA** [comment]
@daren-thomas we merged this a week ago in PR #2225, is the problem still existing?

**Comment by daren-thomas** [comment]
@JIMENOFONSECA confirmed. works in master.
